include/gz/sim/Util.hh

```diff
@@ -267,18 +267,18 @@ namespace gz
         Entity _entity, const EntityComponentManager &_ecm);
 
     /// \brief Environment variable holding resource paths.
-    const std::string kResourcePathEnv{"IGN_GAZEBO_RESOURCE_PATH"};
+    const std::string kResourcePathEnv{"GZ_SIM_RESOURCE_PATH"};
 
     /// \brief Environment variable used by SDFormat to find URIs inside
     /// `<include>`
     const std::string kSdfPathEnv{"SDF_PATH"};
 
     /// \brief Environment variable holding server config paths.
-    const std::string kServerConfigPathEnv{"IGN_GAZEBO_SERVER_CONFIG_PATH"};
+    const std::string kServerConfigPathEnv{"GZ_SIM_SERVER_CONFIG_PATH"};
 
     /// \brief Environment variable holding paths to custom rendering engine
     /// plugins.
-    const std::string kRenderPluginPathEnv{"IGN_GAZEBO_RENDER_ENGINE_PATH"};
+    const std::string kRenderPluginPathEnv{"GZ_SIM_RENDER_ENGINE_PATH"};
```

Comment:
I think this one isn't being ticked either

Reply:
Warning will emit via https://github.com/gazebosim/gz-common/pull/365/commits/21c47bb50f5d3397bfc1c00f647b07ac547d4fc2

https://github.com/gazebosim/gz-sim/pull/1518/commits/ac92c2ca29b222d881e5819611aec99eb26e071f